packages/functional-tests/src/menu.ts

```diff
@@ -220,21 +270,76 @@ export class Menu {
 			}
 		});
 
+		MRE.Actor.Create(this.context, {
+			actor: {
+				name: 'PrevLabel',
+				parentId: this.prevPage.id,
+				transform: {
+					local: {
+						position: { y: buttonWidth * 0.8, z: 0.05 }
+					},
+				},
+				text: {
+					contents: "Prev Page",
+					height: 0.2,
+					anchor: MRE.TextAnchorLocation.BottomCenter
+				}
+			}
+		});
+
+		MRE.Actor.Create(this.context, {
+			actor: {
+				name: 'nextLabel',
+				parentId: this.nextPage.id,
+				transform: {
+					local: {
+						position: { y: buttonWidth * 0.8, z: 0.05 }
+					},
+				},
+				text: {
+					contents: "Next Page",
+					height: 0.2,
+					anchor: MRE.TextAnchorLocation.BottomCenter
+				}
+			}
+		});
+
 		backButton.setBehavior(MRE.ButtonBehavior)
 			.onButton('released', () => {
 				this.back();
 				this.show();
 			});
 
 		this.backActors = [backButton, backLabel];
+	
+		this.nextPage.setBehavior(MRE.ButtonBehavior).onButton('released', () => {
+			if(this.currentNode.menuItems.length > ((this.currentNode.currentPage+1) * pageSize)) {
```

Comment:
>((this.currentNode.currentPage+1) * pageSize) [](start = 42, length = 45)

This expression needs a -1, or else currentPage can == menuItems.length, causing out-of-bounds errors.